src/raft.c

```diff
@@ -485,10 +485,6 @@ static void lockKeys(RedisRaftCtx *rr, raft_entry_t *entry)
 
     for (size_t i = 0; i < num_keys; i++) {
         if (RedisModule_KeyExists(rr->ctx, keys[i])) {
-            size_t str_len;
-            const char *str = RedisModule_StringPtrLen(keys[i], &str_len);
-            LOG_VERBOSE("locking %.*s", (int) str_len, str);
```

Comment:
These logs can help us in debugging issues, we can wrap these lines with a condition in case you care about performance.

Reply:
No, this is not about performance (I think it will also have some performance impact).

Printing log for each key makes log file bigger and makes it hard to analyze.  See the log file here: https://redislabs.atlassian.net/browse/RR-145

It can get worse if we have millions of keys. 

So, if you think it is useful, maybe we should introduce a trace level for it:
https://github.com/RedisLabs/redisraft/blob/52d28196dca6738609825ce08e176db6db2e49ee/src/config.c#L40

Maybe something like TRACE_MIGRATION. Then, probably, it'd be nice to print logs not only when we lock, also when we unlock, import etc maybe?

Reply:
https://github.com/RedisLabs/redisraft/pull/575